<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>词频统计</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
                <widget class="QFrame" name="frame">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                            <horstretch>1</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="text">
                                    <string>可以多选</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QListWidget" name="listWidget">
                                <property name="selectionMode">
                                    <enum>QAbstractItemView::ExtendedSelection</enum>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="frame_2">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                            <horstretch>1</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <widget class="QLabel" name="label_2">
                                <property name="text">
                                    <string>选择一项，显示在表格中</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label_3">
                                <property name="text">
                                    <string>选择多项，只能导出Excel</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QFrame" name="frame_3">
                                <property name="frameShape">
                                    <enum>QFrame::StyledPanel</enum>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Raised</enum>
                                </property>
                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                    <item>
                                        <widget class="QLabel" name="label_4">
                                            <property name="text">
                                                <string>统计阈值&gt;=</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QSpinBox" name="threhold_spinBox"/>
                                    </item>
                                    <item>
                                        <spacer name="horizontalSpacer">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>37</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <widget class="QFrame" name="frame_4">
                                <property name="frameShape">
                                    <enum>QFrame::StyledPanel</enum>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Raised</enum>
                                </property>
                                <layout class="QHBoxLayout" name="horizontalLayout_3">
                                    <property name="spacing">
                                        <number>10</number>
                                    </property>
                                    <property name="leftMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QPushButton" name="btn_ok">
                                            <property name="styleSheet">
                                                <string notr="true">color: rgb(255, 255, 255); background-color: rgb(28,
                                                    177, 245);
                                                </string>
                                            </property>
                                            <property name="text">
                                                <string>OK</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="btn_export">
                                            <property name="text">
                                                <string>导出</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
